c++/src/capnp/persistent.capnp.c++

```diff
@@ -69,8 +69,8 @@ static const ::capnp::_::RawSchema* const d_c8cb212fcd9f5691[] = {
 };
 static const uint16_t m_c8cb212fcd9f5691[] = {0};
 const ::capnp::_::RawBrandedSchema::Dependency bd_c8cb212fcd9f5691[] = {
-  { 33554432,  ::capnp::Persistent< ::capnp::AnyPointer,  ::capnp::AnyPointer>::SaveParams::_capnpPrivate::brand },
-  { 50331648,  ::capnp::Persistent< ::capnp::AnyPointer,  ::capnp::AnyPointer>::SaveResults::_capnpPrivate::brand },
+  { 33554432,  ::capnp::Persistent< ::capnp::AnyPointer,  ::capnp::AnyPointer>::SaveParams::_capnpPrivate::brand() },
```

Comment:
I looked at the disassembly, and dynamic calls *are* generated, unfortunately. Manual inlining will probably be our best option, though I haven't looked into it yet.